pkg/cosign/kms/gcp/gcp.go

```diff
@@ -217,7 +228,17 @@ func (g *KMS) createKey(ctx context.Context) (*ecdsa.PublicKey, error) {
 	}
 	if result, err := g.client.GetCryptoKey(ctx, getKeyRequest); err == nil {
 		fmt.Printf("Key %s already exists in GCP KMS, skipping creation.\n", result.GetName())
-		return g.PublicKey(ctx)
+		k, err := g.PublicKey(ctx)
+		if err != nil {
+			return nil, errors.Wrap(err, "retrieving public key")
+		}
+		pub, ok := k.(*ecdsa.PublicKey)
+		if !ok {
+			if err != nil {
```

Comment:
This looks off a bit, the err has to be nil here because of 232 I think...

Reply:
good catch